compiler/qsc_hir/src/hir.rs

```diff
@@ -185,6 +185,28 @@ impl Display for ItemId {
     }
 }
 
+/// The status of an item.
+#[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
+pub enum ItemStatus {
+    /// The item is defined normally.
+    Normal,
```

Comment:
Personal preference: `Normal` sounds very vague, maybe something more descriptive like `Available`?

Reply:
That sounds reasonable to me! I struggled with what to call this, and forgot to circle back and reevaluate. I like your suggestion better than what I had.